config/locales/en.yml

```
@@ -32,6 +32,8 @@ en:
 
     first_time: First Time? Start Here →
 
+    site_mode: Switch to the full version of the site >>
```

```suggestion
    site_mode: Switch to the full version of the site »
```

Let's use the fancy double chevron unicode.